web/src/main/java/org/mskcc/cbio/oncokb/api/websocket/CurationValidationApiController.java

```diff
@@ -1,6 +1,6 @@
 package org.mskcc.cbio.oncokb.api.websocket;
 
-import com.mysql.jdbc.StringUtils;
```

Comment:
Noticed that we use StringUtils from mysql instead of apache in a couple of files.

Reply:
Thanks, we can migrate when possible.